frontend-mini-challenge/components/LoggedInPage.tsx

```diff
@@ -131,13 +145,17 @@ Then when you make use of this page component:
     </Page>
 
 */
-export async function getServerSidePropsForLoggedInPage({ query, req }) {
-  const whoAmIResponse = await serverSideWhoAmI({ query, req });
+
+
+export const getServerSidePropsForLoggedInPage = async (context: GetServerSidePropsContext) => {
```

Comment:
explicit is better than implicit. 

Are we really expecting the props to look like [this](https://github.com/vercel/next.js/blob/v10.2.0/packages/next/types/index.d.ts#L136-L151)?

It looks like we don't use most of that.

Good code makes its intentions clear. So it's important to be specific about what is required.

We are using Typescript so that when other coders work on this project they don't mess up and pass the wrong thing into a function. So we should not accept the wrong thing. We should only accept exactly the correct thing.